### DISCUSSION TITLE
Github copilot with github student just ended

### DISCUSSION THREAD

**Discussion by benja971** [community/community#41376] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hello everyone, 

i'm reposting this topic because all of the ones I have seen are closed. I'm having a github student verified account and my github account linked to my VScode but I'm getting the folling error :

![image](https://user-images.githubusercontent.com/46868460/206879964-847104c5-ff63-4094-9f49-7c3f908374ea.png)

I've tried to reload vscode, disconnect and reconnect github, uninstall and reinstall github copilot extension unsuccesfully.

Could someone have an explanation ?

thank you in advance

**Comment by iamcjwilliams** [comment]
@benja971 Sorry for the trouble here, Have you tried to verify your account [here](https://education.github.com/discount_requests/pack_application) in order to gain access?

**Reply by bilalnaeemcs** [reply]
I was also facing the same problem, I reverified, but I am still facing the same problem!
